src/transformers/trainer.py

```diff
@@ -1641,6 +1647,7 @@ def _inner_training_loop(
 
         self.state = TrainerState()
         self.state.is_hyper_param_search = trial is not None
+        self.state.train_batch_size = self._train_batch_size
```

Comment:
Won't this mean the batch_size from the state is always loaded even if `self.args.auto_find_batch_size` is `False` ?

Reply:
Yes, we care about if it was called *at all* and the metadata exists inside the state.